src/main/java/org/apache/commons/lang3/builder/ReflectionToStringBuilder.java

```diff
@@ -867,4 +874,21 @@ private void validate() {
         }
     }
 
+    private boolean handleNativeClasses() {
+        Object value = getObject();
+        if (value instanceof Number || value instanceof Boolean || value instanceof Character
+            || value instanceof TemporalAccessor ) {
+            getStringBuffer().append("value=").append(getObject().toString());
+            return true;
+        }
+        
+        if (value.getClass().isArray() || value instanceof Collection || value instanceof Map) {
```

Comment:
Can `value` be null here?

Reply:
nope, that is covered in line 846 of toString()